src/main/java/com/google/api/generator/engine/ast/JavaDocComment.java

```diff
@@ -92,7 +92,7 @@ public Builder addSampleCode(String sampleCode) {
       Arrays.stream(sampleCode.split("\\r?\\n"))
           .forEach(
               line -> {
-                componentsList.add(line);
+                componentsList.add(line.replace("@Override", "{@literal @}Override"));
```

Comment:
Is this meant to be temporary? Shouldn't we handle `@` in general, for annotations that are not `@Override`?

Reply:
It is temporary solution.

Reply:
Can we add a TODO here?

Reply:
I have problem here. If we handle  `@` in  annotations `JavaWriteVistor`, the `google-java-formatter` will complain an error `com.google.googlejavaformat.java.FormatterException: 3:13: error: <identifier> expected`

`JavaWriterVisitor` looks like 
```
  @Override
  public void visit(AnnotationNode annotation) {
    if (!annotation.escapeCharacterAt()) {
      buffer.append(AT);
    } else {
      leftBrace();
      buffer.append(AT);
      buffer.append(LITERAL);
      space();
      buffer.append(AT);
      rightBrace();
    }
    annotation.type().accept(this);
    if (annotation.descriptionExpr() != null) {
      leftParen();
      annotation.descriptionExpr().accept(this);
      rightParen();
    }
    newline();
  }
```

In this case, we have to format the code first, and then escape the character `@`. What do you think? Thank @miraleung

Reply:
Will add `todo` to mention it is the temporary solution, and will discuss the long-term solution off channel.